gallery/visualtests/nxRadio.js

```diff
@@ -76,5 +77,17 @@ describe('NxRadio', function() {
     it('looks disabled when hovered', hoverTest(disabledCheckedSelector));
   });
 
+  describe('Tooltip for NxRadio', function() {
+    it('has a tooltip on hover when set to true', async function() {
```

Comment:
It seems like this could have been done with `hoverTest()`.  Notice its optional third parameter which is for this case.

Reply:
> It seems like this could have been done with `hoverTest()`. Notice its optional third parameter which is for this case.

I didn't know that I could have two selectors and wait for the tooltip, thanks! 28e27ee9fdb3d586de9a5536e59d0f52643e0883